lib/RoadizCoreBundle/src/Repository/NodeTypeFilesRepository.php

```diff
@@ -155,6 +157,10 @@ private function deserialize(string $content): NodeType
             throw new \RuntimeException('Deserialized NodeType is not an instance of NodeType');
         }
 
+        if (strtolower($nodeType->getName()) !== explode('.', $fileName)[0]) {
```

Comment:
```suggestion
        if (mb_strtolower($nodeType->getName()) !== explode('.', $fileName)[0]) {
```

Use whenever possible `mb_` transform method to deal with non-ascii chars.